manifests/spel-minimal-centos-7-virtualbox.manifest.txt

```diff
@@ -1,74 +1,72 @@
 acl-2.2.51-14.el7.x86_64
-audit-2.8.4-4.el7.x86_64
-audit-libs-2.8.4-4.el7.x86_64
+audit-2.8.5-4.el7.x86_64
+audit-libs-2.8.5-4.el7.x86_64
 authconfig-6.2.8-30.el7.x86_64
 autogen-libopts-5.18-5.el7.x86_64
 basesystem-10.0-7.el7.centos.noarch
-bash-4.2.46-31.el7.x86_64
-bind-libs-lite-9.9.4-73.el7_6.x86_64
-bind-license-9.9.4-73.el7_6.noarch
-binutils-2.27-34.base.el7.x86_64
+bash-4.2.46-33.el7.x86_64
+bind-export-libs-9.11.4-9.P2.el7.x86_64
+binutils-2.27-41.base.el7_7.3.x86_64
 btrfs-progs-4.9.1-1.el7.x86_64
 bzip2-libs-1.0.6-13.el7.x86_64
-ca-certificates-2018.2.22-70.0.el7_5.noarch
-centos-release-7-6.1810.2.el7.centos.x86_64
+ca-certificates-2019.2.32-76.el7_7.noarch
+centos-release-7-7.1908.0.el7.centos.x86_64
```

Comment:
I guess also note that the vagrant box is now CentOS 7.7, but this might need some investigation as that should have updated a loooong time ago!

Reply:
Added note